plover/oslayer/osxkeyboardcontrol.py

```diff
@@ -289,7 +293,7 @@ def _event_handler(self):
 
 # "Narrow python" unicode objects store characters in UTF-16 so we
 # can't iterate over characters in the standard way. This workaround
-# let's us iterate over full characters in the string.
+# lets us iterate over full characters in the string.
 def characters(s):
```

Comment:
We should probably move this to the newly added `plover.misc` (and remove the duplicate in `winkeyboardcontrol`).